css/index.css

```diff
@@ -1 +1,493 @@
-/* Use your CSS from RWD - I here.  IF you don't have CSS, use the solution CSS from RWD - I*/
\ No newline at end of file
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+    font-size: 62.5%;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+.testing * {
+    border: 1px solid palevioletred;
+}
+
+.container {
+  max-width: 880px;
+  /* TB LR */
+  margin: 0 auto;
+}
+
+/* header */
+
+header {
+    display: flex;
+    width: 100%;
+}
+
+header nav {
+  display: flex;
+  width: 100%;
+  vertical-align: top;
+  margin-top: 39px;
+  justify-content: space-between;
+  flex-wrap: nowrap;
+  margin-right: 92px;
+}
+
+header nav a {
+  display: inline-block;
+  color: gray;
+  text-decoration: none;
+  font-size: 1.5rem;
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+header .logo_container {
+  display: flex;
+  margin-top: 25px;
+  width: auto;
+  margin-right: 2px;
+}
+
+.logo_container img{
+	height: 100%;
+	width: auto;
+}
+
+/* cta */
+
+.cta {
+
+}
```

Comment:
You have an empty selector (`.cta`); that's no good!

Reply:
Don't know how that one slipped by me.